maestro-orchestra/src/main/java/maestro/orchestra/workspace/WorkspaceExecutionPlanner.kt

```diff
@@ -119,6 +146,18 @@ object WorkspaceExecutionPlanner {
         return null
     }
 
+    private fun isFlowFile(path: Path): Boolean {
+        if (!path.isRegularFile()) return false // Not a file
+        val extension = path.extension
+        if (extension != "yaml" && extension != "yml") return false // Not YAML
+        if (path.nameWithoutExtension == "config") return false // Config file
+        return true
```

Comment:
Can we have different error messages for the wrong extension or missing extension as well?
If the flow isn't found because of a type in the extension it can be hard to find the reason, especially on Windows where extensions are usually not shown to the user.


You can ignore this comment if you think this is out of the scope of this PR.

Reply:
Hmm the workspace may have other types of files so we can't know whether a non-yaml extension is a mistake or not